src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -72,9 +72,9 @@ The message bundle is attached to each panel component as the `parentBundle` opt
 Message bundles can be specified in one of two ways:
 
 1. through the [auxiliary schema](AuxiliarySchemaForPreferencesFramework.md) (if schemas are being used), or
-1. directly to the `messageLoader` (if grades are being used).
+2. directly to the `messageLoader` (if grades are being used).
 
-The Preferences Framework will load and combine all of the Message Bundles into a single Message Bundle which is bound to the `prefsEditorLoader` component at the `msgResolver` property (as described above).
+The Preferences Framework will load all of the Message Bundles. When using an auxiliary schema the bundles are automatically distributed to the panels. If an auxiliary schema is not used, the `messageBase` options of each panel will need to be set manually using an IoC reference; `{messageLoader}.resources.<messageBundle>.resourceText`.
 
 Any panel that has the grade `fluid.prefs.defaultPanel` will have access to the combined Message Bundle at its `parentBundle` option (as described above). When using the auxiliary schema, all panels are assigned the grade `fluid.prefs.defaultPanel` by the Framework.
```

Comment:
looks like `fluid.prefs.defaultPanel` doesn't exist anymore at all, so i'll just remove that whole section.